### ISSUE TITLE
Unable to open file (Unable to open file: name = 'apnamodel.best.hdf5', errno = 2, error message = 'no such file or directory', flags = 0, o_flags = 0)

### ISSUE THREAD

**Issue by Ham72** [keras-team/keras#8661]
I am trying to save best weights of Convoluted NN using Keras, and want to evaluate model based on best weights. But the file for saving best weight does not create.  I have given absolute path, changed the name of file but nothing worked. Although the file is not created in the working directory, error occurs at line "model.load_weights("apnamodel.best.hdf5")"

Here is the code

model = Sequential() 
model.add(Conv2D(32, 2, strides=1, activation='relu', input_shape = input_shape)) 
model.add(MaxPooling2D(pool_size=2))
model.add(Flatten())
model.add(Dense(50, activation='relu'))
model.add(Dense(10, activation= "softmax"))
nadam  =  optimizers.Nadam(lr=0.002, beta_1=0.9, beta_2=0.999, epsilon=1e-08, schedule_decay=0.004)
model.compile(loss='categorical_crossentropy', optimizer=nadam, metrics=['accuracy'])

# checkpoint

filepath="apnamodel.best.hdf5"
checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=0, save_best_only=True, mode='max')
callbacks_list = [checkpoint]

earlystop = EarlyStopping(monitor='val_loss', min_delta=0.0001, patience=3, verbose=0, mode='auto')
callbacks_list = [earlystop]

history=model.fit(train_data, label_train_tranformed, epochs=5, batch_size=1000, validation_data=(val_data, label_val_tranformed), callbacks=callbacks_list)
score = model.evaluate(test_data, label_test_tranformed, batch_size=100)
print("score1 :"+ str(score[0]))
print("score2 :"+ str(score[0]))

model = Sequential() 
model.add(Conv2D(32, 2, strides=1, activation='relu', input_shape = input_shape)) 
model.add(MaxPooling2D(pool_size=2))
model.add(Flatten())
model.add(Dense(50, activation='relu'))
model.add(Dense(10, activation= "softmax"))
model.load_weights("apnamodel.best.hdf5")
nadam  =  optimizers.Nadam(lr=0.002, beta_1=0.9, beta_2=0.999, epsilon=1e-08, schedule_decay=0.004)
model.compile(loss='categorical_crossentropy', optimizer=nadam, metrics=['accuracy'])
scores = model.evaluate(val_data, label_val_tranformed)
print ("Scores "+str(scores))

**Comment by alanlivio** [comment]
Just to registred here I was able to bypass the error by using the variable HDF5_USE_FILE_LOCKING='FALSE' from the command line.
